pkg/nodeconfig/nodeconfig.go

```diff
@@ -264,6 +266,14 @@ func (nc *nodeConfig) Configure() error {
 			return errors.Wrapf(err, "error updating version annotation on node %s", nc.node.GetName())
 		}
 
+		// If we deploy on Azure with CCM support, we have to explicitly remove the node taint here, because the node manager
+		// running on the node can't do it itself.
+		if azureCloudNodeManagerConfigured {
+			if err := nc.excludeCloudTaint(); err != nil {
+				return errors.Wrapf(err, "error excluding cloud taint on node %s", nc.node.GetName())
+			}
+		}
```

Comment:
This would make more sense to me if it was called within the `configureAzureCloudNodeManager` function, as it seems as if this taint removal is essentially indicating to the cluster that the service is running?

Reply:
To be clear, nodeConfig.configureAzureCloudNodeManager